package.json

```diff
@@ -24,15 +24,15 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "ejs": "1.0.x",
+    "ejs": "^2.2.4",
     "express": "2.5.x",
```

Comment:
I would prefer if we go for x-ranges matching major and minor versions (in this case "ejs": 2.2.x)